packages/editor/src/client/panel/publishPagePanel.tsx

```diff
@@ -12,27 +12,32 @@ import {InfoColor} from '../atoms/infoMessage'
 import {DescriptionListItemWithMessage} from '../atoms/descriptionListwithMessage'
 
 export interface PublishPagePanelProps {
-  initialPublishDate?: Date
+  publishedAtDate?: Date
+  updatedAtDate?: Date
+  publishAtDate?: Date
   pendingPublishDate?: Date
   metadata: PageMetadata
 
   onClose(): void
-  onConfirm(publishDate: Date, updateDate: Date): void
+  onConfirm(publishedAt: Date, updatedAt: Date, publishAt: Date): void
 }
 
 export function PublishPagePanel({
-  initialPublishDate,
+  publishedAtDate,
+  updatedAtDate,
+  publishAtDate,
   pendingPublishDate,
   metadata,
   onClose,
   onConfirm
 }: PublishPagePanelProps) {
   const now = new Date()
 
-  const [publishDate, setPublishDate] = useState<Date | undefined>(
-    pendingPublishDate ?? initialPublishDate ?? now
-  )
-  const [updateDate, setUpdateDate] = useState<Date | undefined>(now)
+  const [publishedAt, setPublishedAt] = useState<Date | undefined>(publishedAtDate ?? now)
```

Comment:
I think you still need the pending date, but if I am following right it needs to be in the publishAt state.